sys/kern/tmpfs.c

```diff
@@ -111,6 +115,12 @@ typedef struct tmpfs_node {
   };
 } tmpfs_node_t;
 
+enum tmpfs_time_type {
```

Comment:
I think the code style of `mimiker/sys/kern` directory would suggests here `typedef enum` instead of just `enum`.

Reply:
good remark